arc/src/main.c

```diff
@@ -1089,20 +1101,25 @@ static void handle_pme(struct zjs_ipm_message* msg)
             DBG_PRINT("%d ", msg->data.pme.vector[i]);
         }
 
-        curie_pme_write_vector(msg->data.pme.vector, msg->data.pme.vector_size);
-        DBG_PRINT("\nwrote with %d neruons\n", curie_pme_get_committed_count());
+        curie_pme_write_vector(msg->data.pme.vector,
+                               msg->data.pme.vector_size);
+        DBG_PRINT("\nwrote with %d neruons\n",
+                  curie_pme_get_committed_count());
         break;
     case TYPE_PME_GET_COMMITED_COUNT:
         msg->data.pme.committed_count = curie_pme_get_committed_count();
-        DBG_PRINT("committed count: %d\n", msg->data.pme.committed_count);
+        DBG_PRINT("committed count: %d\n",
+                  msg->data.pme.committed_count);
         break;
     case TYPE_PME_GET_GLOBAL_CONTEXT:
         msg->data.pme.g_context = curie_pme_get_global_context();
-        DBG_PRINT("global context: %d\n", msg->data.pme.g_context);
+        DBG_PRINT("global context: %d\n",
+                  msg->data.pme.g_context);
         break;
     case TYPE_PME_SET_GLOBAL_CONTEXT:
         // valid range is 1-127
-        if (msg->data.pme.g_context < 1 || msg->data.pme.g_context > 127) {
+        if (msg->data.pme.g_context < 1 ||
+            msg->data.pme.g_context > 127) {
```

Comment:
I really don't think this is more readable either, multi-line if conditions are best avoided...